packages/dd-trace/test/appsec/remote_config/index.spec.js

```diff
@@ -158,4 +162,73 @@ describe('Remote Config enable', () => {
       })
     })
   })
+
+  describe('ASM_DD remote config', () => {
+    describe('listener', () => {
+      let listener
+
+      beforeEach(() => {
+        config.appsec = { enabled: undefined }
+
+        remoteConfig.enable(config)
+        remoteConfig.enableAsmDDRules(config.appsec)
+        listener = rc.on.secondCall.args[1]
+      })
+
+      it('should call RuleManager.updateAsmDDRules', () => {
+        const rules = {}
+        listener('apply', rules)
+        expect(RuleManager.updateAsmDDRules).to.have.been.calledOnceWithExactly('apply', rules)
+      })
+    })
+
+    describe('enable', () => {
+      it('should not fail if remote config is not enabled before', () => {
+        config.appsec = {}
+        remoteConfig.enableAsmDDRules(config.appsec)
+        expect(rc.on).to.not.have.been.calledWith('ASM_DD')
+      })
+
+      it('should not activate if rules is configured', () => {
+        config.appsec = { enabled: true, rules: './path/rules.json' }
+        remoteConfig.enable(config)
+        remoteConfig.enableAsmDDRules(config.appsec)
+        expect(rc.on).to.not.have.been.calledWith('ASM_DD')
+      })
+
+      it('should activate if appsec is manually enabled', () => {
+        config.appsec = { enabled: true }
+        remoteConfig.enable(config)
+        remoteConfig.enableAsmDDRules(config.appsec)
+        expect(rc.on).to.have.been.calledOnceWith('ASM_DD')
+      })
+
+      it('should activate if appsec enabled is not defined', () => {
+        config.appsec = {}
+        remoteConfig.enable(config)
+        remoteConfig.enableAsmDDRules(config.appsec)
+        expect(rc.on.lastCall).to.have.been.calledWith('ASM_DD')
+      })
+    })
+  })
+
+  describe('disableAsmData', () => {
```

Comment:
I would put the `disableAsmData` and `DisableAsmDDRules` tests under the `ASM_DD remote config` describe() like:
```javascript
describe('ASM_DD remote config', () => {
  describe('listener', () => {})
  describe('enable', () => {})
  describe('disable', () => /* put it here */)
})
```